Readme.md

```diff
@@ -222,60 +222,113 @@ if(maybeABasket.isNothing()) {
 ## Maybe api
 
 - Maybe take a value and return a maybe, if the passed value is null or undefined, all operation on the maybe to change the value will be ignored.
```

Comment:
Maybe takes a value and returns a maybe. When the value given in argument is null or undefined, all the operations applied to the maybe monad to change its value will be ignored.